packages/core/strapi/lib/core/registries/apis.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
 const { has } = require('lodash/fp');
-const { createCoreApi } = require('../../core-api');
+const { createService } = require('../../core-api/service');
+const createController = require('../../core-api/controller');
```

Comment:
We might consider unifying the way we request `createService` & `createController`?